cluster/manifests/skipper/deployment.yaml

```diff
@@ -79,7 +79,7 @@ spec:
           whenUnsatisfiable: DoNotSchedule
           labelSelector:
             matchLabels:
-              deployment: skipper-ingress
```

Comment:
Should we add application label as well?

Reply:
good idea!